students/dychowicz_monika/lesson_04_unit_testing/lesson_04_unit_testing.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+
+
+def fib(n):
+    if not isinstance(n, int):
+        raise TypeError("Enter the integer number")
+    elif n < 0:
+        raise ValueError("Enter a positive integer number")
+    elif n == 1 or n == 2:
+        return 1
+    elif n == 0:
+        return 0
+    else:
+        return fib(n - 1) + fib(n - 2)
+
+    class TestClass(object):
```

Comment:
No need to inherit from `object`
Wrong class name. It does not represent anything.